content/_policy/contact-submitted._en.md

```diff
@@ -0,0 +1,15 @@
+---
+permalink: /contact/case-submitted/
```

Comment:
Could we align the permalink with the filename? One is `case-submitted` and the other is `contact-submitted`